LICENCE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2007-2023 Nils Anders Danielsson, Ulf Norell, Shin-Cheng
+iCopyright (c) 2007-2024 Nils Anders Danielsson, Ulf Norell, Shin-Cheng
```

Comment:
What's with the `i`?

```suggestion
Copyright (c) 2007-2024 Nils Anders Danielsson, Ulf Norell, Shin-Cheng
```

Reply:
I think that's a remnant of me using Emacs for the first time after using Vim for a while... Fixed.